_zip/missingapi.yml

```diff
@@ -25,6 +25,22 @@ references:
   name: GetListPadInfo(String, MMC_LISTPAD_INFO)
   fullname: Microsoft.CLRAdmin.IExtendTaskPad.GetListPadInfo(String, MMC_LISTPAD_INFO)
   href: https://docs.microsoft.com/previous-versions/dotnet/netframework-1.1/clr-admin/5stkefbf(v=vs.90)
+- uid: Polly.Registry.PolicyRegistry
+  name: PolicyRegistry
+  fullname: Polly.Registry.PolicyRegistry
+  href: https://github.com/App-vNext/Polly/wiki
+- uid: Polly.Registry.IPolicyRegistry%601
+  name: IPolicyRegistry<TKey>
+  fullname: Polly.Registry.IPolicyRegistry<TKey>
+  href: https://github.com/App-vNext/Polly/wiki
+- uid: Polly.Registry.IReadOnlyPolicyRegistry%601
+  name: IReadOnlyPolicyRegistry<TKey>
```

Comment:
This is identical to line 41 AFAICT. Is it supposed to be ReadOnlyPolicyRegistry<TKey>?

Reply:
Thanks for catching. It shouldn't not have been added. There were only 3 types generating warnings.